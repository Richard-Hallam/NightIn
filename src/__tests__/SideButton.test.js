import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import SideButton from '../components/matchButtons';

test('SideButton component renders correctly', () => {
  const { container } = render(<SideButton />);
  expect(container).toMatchSnapshot();
});