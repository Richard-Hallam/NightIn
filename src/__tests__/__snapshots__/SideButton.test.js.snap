// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideButton component renders correctly 1`] = `
<div>
  <div
    class="button-container"
  >
    <button
      class="side-button"
    >
      <span
        class="material-symbols-outlined"
        style="font-size: 100pt;"
      >
        person_add
      </span>
    </button>
    <button
      class="match-button"
    >
      let's watch.
    </button>
  </div>
</div>
`;
